### ISSUE TITLE
wrong slot start time for next days

### ISSUE THREAD

**Issue by nsocheleau** [fullcalendar/fullcalendar-scheduler#221]
Hello,

I'm trying to setup a schedule view with 2 time slots per day (5h slots) starting at 7am.
For the first day in the view, it shows 2 columns with 7am and 12pm es expected.
For the next day, the columns are 8am and 1pm. For the 3rd, 9am and 2pm.

http://jsbin.com/worusuwiju/edit?js,output

I am expecting 7am and 12pm columns for the days.

Is there a setting that I am missing?

Thanks,
Nicolas

**Comment by arshaw** [comment]
This looks like a bug. I would have expected the same. I will investigate.

**Comment by nsocheleau** [comment]
Do you have some documentation on how to run the tests and contribute test cases?

**Comment by arshaw** [comment]
yes! these docs are written for the main fullcalendar project, but it's a similar setup for scheduler (except with coffeescript). Also, the gulp task names are different (would be `gulp karma` instead of `gulp test`)

**Comment by arshaw** [comment]
the link: https://github.com/fullcalendar/fullcalendar/wiki/Automated-Tests
